### PR TITLE
clarify experimental wording

### LINKED ISSUE #22391: Clarify confusing wording in MDN "Experimental" definition
### MDN URL

https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete

### What specific section or headline is this issue about?

"Experimental"

### What information was incorrect, unhelpful, or incomplete?

From @wolenetz: "https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete#experimental could probably use clarification around the meaning of "It is implemented and enabled by default in less than two modern major browsers" -- I understand that it is being applied as if s/major browsers/major distinct underlying rendering engines in browsers/, but on the surface it is confusing."

### What did you expect to see?

I think the fix here is probably to update the "two modern major browsers" to talk about rendering engines instead (two instances), and also to change the description around specification status, e.g. "A feature or technology that is not marked experimental or deprecated is said to be on a standards track" - quite often features only in Chrome are defined in stable specifications.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### PR DESCRIPTION
**Pull request by chrisdavidmills**
### Description

As per https://github.com/mdn/content/issues/22391, some issues were brought up with the wording on the MDN "experimental" definition. This PR aims to fix that.

### Related issues and pull requests

Fixes https://github.com/mdn/content/issues/22391

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/experimental_deprecated_obsolete/index.md [hunk 1]
```diff
@@ -18,20 +18,20 @@ When a technology is described as experimental on MDN Web Docs, it means that th
...
-Conversely, a technology is no longer considered **experimental** if one of the following conditions is true:
+Conversely, a technology is no longer considered **experimental** if one or more of the following conditions is true:
 
-- It is implemented in two or more major browsers.
+- It is implemented in two or more major browser rendering engines.
 - Its defining specification is unlikely to change in ways that will break the web.
 
 Usually, if a technology is supported across several major browsers, the specification will be stable, but this is not always the case.
-And some technologies might have a stable specification and be well-used but might not have native support in browsers ([IMSC](/en-US/docs/Related/IMSC), for example). <!-- need to revisit link -->
+And some technologies might have a stable specification and be well-used, but not have native support in browsers ([IMSC](/en-US/docs/Related/IMSC), for example). <!-- need to revisit link -->
```
Comment (hamishwillee) [inline files/en-us/mdn/writing_guidelines/experimental_deprecated_obsolete/index.md:32]: Don't know what "not have native support in browsers" means even after going to IMSC. How can something be "well used" but not supported?

I'd also start this sentence with "Note that" ...

Comment (queengooborg) [reply files/en-us/mdn/writing_guidelines/experimental_deprecated_obsolete/index.md:32]:
```suggestion
On the other hand, some technologies might have a stable specification, but do not have native support in browsers ([IMSC](/en-US/docs/Related/IMSC), for example). <!-- need to revisit link -->
```

Comment (hamishwillee) [reply files/en-us/mdn/writing_guidelines/experimental_deprecated_obsolete/index.md:32]: @chrisdavidmills ^^^ that suggestion works for me.

**Review by queengooborg — APPROVED** [review]
Thanks for tackling this, LGTM!

**Pull request merged**